### PR TITLE
rubocop: Naming/PredicateName: Add is_to_s to AllowedMethods

### PR DESCRIPTION
**Pull request by root-expert**

### CODE REVIEW

**Review by ekohl — APPROVED** [review]
Looks good. Are there more we should capture?

**Comment by root-expert** [comment]
> Looks good. Are there more we should capture?

I was wondering the same. Nothing coming up on my mind, maybe someone else has an idea

File: rubocop.yml [hunk 1]
```diff
@@ -439,6 +439,7 @@ Style/PerlBackrefs:
 
 Naming/PredicateName:
   Enabled: True
+  AllowedMethods: ['is_a', 'is_to_s']
```
Comment (alexjfisher) [inline rubocop.yml:442]: Shouldn't this have been...
```suggestion
  AllowedMethods: ['is_a?', 'is_to_s']
```

Comment (root-expert) [reply rubocop.yml:442]: Hmm, yea the default was with ?
I'll push another PR and fix it

**Pull request merged**